pkg/watch/mux.go

```diff
@@ -162,6 +165,7 @@ func (m *Broadcaster) loop() {
 		}
 		m.distribute(event)
 	}
+	m.distributing.Done()
```

Comment:
Which would result in closing the channels before we waited for all the info to have been sent to them. I don't see why.